ci-operator/step-registry/image-based-install-operator/reinstall/create-secrets/image-based-install-operator-reinstall-create-secrets-commands.sh

```diff
@@ -0,0 +1,102 @@
+kind: Secret
+metadata:
+  name: ibi-cluster-admin-kubeconfig
+  namespace: ibi-cluster
+  labels:
+    image-based-installed.openshift.io/created: "true"
+type: Opaque
+EOSEC
+
+tee <<EOSEC | oc create -f -
+apiVersion: v1
+data:
+  password: SUFnSE4tYXZMWHAtc3YyWXEtN3hVVTg=
+  username: a3ViZWFkbWlu
+kind: Secret
+metadata:
+  name: ibi-cluster-admin-password
+  namespace: ibi-cluster
+  labels:
+    image-based-installed.openshift.io/created: "true"
+type: Opaque
+EOSEC
+
+pull_secret=$(cat $PULL_SECRET_FILE)
```

Comment:
Add a comment explaining how this was generated (in case we need to update  it in the future)